packages/runtime-common/error.ts

```diff
@@ -137,7 +137,7 @@ export function responseWithError(error: CardError): Response {
     {
       status: error.status,
       statusText: error.title,
-      headers: { 'content-type': 'application/vnd.api+json' },
+      headers: { 'content-type': 'application/vnd.card+json' },
```

Comment:
Should this change or stay the same? I kind of feel like it should be `application/json` /cc @habdelra

Reply:
agreed, this is not a card response

Reply:
Updated.